tests/end2end/test_custom_op.py

```diff
@@ -45,6 +45,6 @@ def test_custom_op(self):
         actual = model.predict(x)
         self.assertIsNotNone(actual)
 
-        converted_model = onnxmltools.convert_keras(model, custom_conversions={ScaledTanh: custom_activation})
+        converted_model = onnxmltools.convert_keras(model, custom_conversion_functions={ScaledTanh: custom_activation})
         self.assertIsNotNone(converted_model)
-        # print(str(converted_model))
+        print(str(converted_model))
```

Comment:
We are we printing it? Better to check the # of nodes and the node name if you want to double-check the outcome.